### ISSUE TITLE
Change-of-owner for Quality Bionics

### ISSUE THREAD

**Issue by Vectorial1024** [Vectorial1024/EliteBionicsFramework#79]
Quality Bionics will be transferred to the community mod steward Mlie after original owner became unable to provide a 1.5 update.

The situation is currently still developing.

**Comment by Vectorial1024** [comment]
New Quality Bionics seem to have problems with EBF.

**Comment by Vectorial1024** [comment]
For convenience, their 1.5 version is here: https://steamcommunity.com/sharedfiles/filedetails/?id=3236892428

**Comment by Vectorial1024** [comment]
For some reason the updated Quality Bionics walled off the very important settings class, and EBF can't do anything about it.

Negotiating.

**Comment by Vectorial1024** [comment]
Note: if this negotiation turns south, and not even Community Unification (which itself took quite some effort) is enough to make Quality Bionics (or any of its continuation mods) work with other bionic mods, then we EBF may respond to this with even greater strength.

**Comment by Vectorial1024** [comment]
Negotiation concluded; essentially, Quality Bionics (Continued) changed some of its namespace (?!), so some patches worked but some didn't. And the ones that did not work were the ones that I did not expect to not work.

A confusing move by the new maintainer when all that had to be done was to rebuild to make it work in 1.5, to break continuity of systems for no obvious immediate reason (they did mention they have some new plans for it, but after this exchange, better release the new plans under a new name).

We will still update the Quality Bionics compatibility.

---

PS: consider how I renamed ClosedDoors to Desynchronized when I took it over a few years ago when I have a new plan in my mind.
